challenges.html

```diff
@@ -9,7 +9,7 @@
     <title>[ESC] Hacker Escape Rooms</title>
 </head>
 
-<body>
+<body onload="wellHello()">
```

Comment:
It's better to put wellHello() in the displayChallenges.js files since it's loaded from line 132:
<script src="displayChallenges.js"></script>

Reply:
Noted!